prime-router/src/main/kotlin/azure/HttpUtilities.kt

```diff
@@ -308,5 +312,157 @@ class HttpUtilities {
                 return responseCode to response
             }
         }
+
+        /**
+         * A generic function to GET data to a particular Prime ReportStream Environment,
+         * as if from Organization.
+         * Returns Pair(Http response code, json response text)
+         */
+        fun getOrganization(
```

Comment:
This is not the place for this method.  This class is for generic HTTP utilities.  Move this to a different class or a new class.